docs/integrations/creating-nodes/build/reference/ui-elements.md

```diff
@@ -732,4 +732,72 @@ Display a yellow box with a hint or extra info. Refer to [Node UI design](/integ
   default: '',
 },
 ```
-![Notice](/_images/integrations/creating-nodes/notice.png)
\ No newline at end of file
+![Notice](/_images/integrations/creating-nodes/notice.png)
+
+## Hints
+
+There are two types of hints: input hints and larger hints.
```

Comment:
I would say it mostly differ not in size but in where they are displayed, so parameters hints and node hints